daemon/daemon.go

```diff
@@ -217,5 +219,7 @@ func main() {
 
 		fmt.Println("==== Start Management portal if not running")
 		managementServerUp()
+		//TODO POC avahi
+		avahi.InitMDNS()
 	}
```

Comment:
should this be moved into the managementServerUp() method, particular inside the code block that only runs when the management server is not already up: https://github.com/CanonicalLtd/UCWifiConnect/blob/master/daemon/daemon.go#L99 ?